### ISSUE TITLE
Undefined Model names

### ISSUE THREAD

**Issue by theanubhavk** [RockChinQ/free-one-api#20]
What should be the model names if I'm using hugging face or bard ?

**Comment by RockChinQ** [comment]
This parameter is not considered now, you can directly set the model as `gpt-3.5-turbo`.

**Comment by egcash** [comment]
Yeah,I am now using this model,but what would be the model name If i created channel for bard or hugging chat?

> model

**Comment by RockChinQ** [comment]
> Yeah,I am now using this model,but what would be the model name If i created channel for bard or hugging chat?
> 
> > model

In fact, hugchat's model may be LLaMA, but the adapters for hugchat and bar in the Free One API do not consider the model required in the request, but instead directly use the default configuration parameters provided by the upstream reverse engineering library.

**Comment by RockChinQ** [comment]
Support for switching models may be added in future versions.
